sunpy/net/attr.py

```diff
@@ -60,6 +113,55 @@ def collides(self, other):
     def __eq__(self, other):
         return dict(vars(self)) == dict(vars(other))
 
+    @classmethod
+    def update_values(cls, adict):
+        """
+        This is the method that clients will use to register their values for subclasses of `Attr`.
+        The input should be a dictionary.
+        Each key should be a subclass of Attr.
+        The value for each key should be a list of tuples.
+        The tuple should be of the form (`Name`, `Description`).
+        If you do not want to add a description, you can put None or "".
+        We sanitize the name you provide by removing all special characters and making it all lower case.
+        If it still isn't valid we will append to the start of the name to make it a valid attribute name.
+
+        **It is recommended that clients register values for all attrs used by their `_can_handle_query` method.**
+
+        We have an example here for an Instrument Class.
+
+        Example
+        -------
+        >>> from sunpy.net import attr, attrs
+        >>> attr.Attr.update_values({attrs.Instrument: [('AIA', 'AIA is in Space.'), ('HMI', 'HMI is next to AIA.')]})
+        >>> attr.Attr._attr_registry[attrs.Instrument]
+        attr(name=['aia', 'hmi'], name_long=['AIA', 'HMI'], desc=['AIA is in Space.', 'HMI is next to AIA.'])
+        >>> attr.Attr._attr_registry[attrs.Instrument].name
+        ['aia', 'hmi']
+        >>> attr.Attr._attr_registry[attrs.Instrument].name_long
+        ['AIA', 'HMI']
+        >>> attr.Attr._attr_registry[attrs.Instrument].desc
+        ['AIA is in Space.', 'HMI is next to AIA.']
+        >>> attrs.Instrument.aia, attrs.Instrument.hmi
+        (<Instrument('AIA')>, <Instrument('HMI')>)
+        """
+        for k, v in adict.items():
+            if isiterable(v) and not isinstance(v, str):
+                for pair in v:
+                    if len(pair) != 2:
+                        raise ValueError('Please check your input dictionary, \
```

Comment:
This error should tell you what key it fails on. i.e. `"Invalid input key 'Instrument' is not an interable of pairs."`